src/index.js

```diff
@@ -15,9 +15,18 @@ if (process.env.NODE_ENV !== "production") {
 
 const mapConfig = parseUriHash(defaultMapConfig);
 
-new LeafletMap(mapConfig);
+const mapContainer = new LeafletMap(mapConfig);
 new InfoWindow();
 new TitleDetails();
 new LoadingIndicator();
 
-getData();
+const {
+  _northEast: topRight,
+  _southWest: bottomLeft,
+} = mapContainer.map.getBounds();
+
+// Only load data in the map view
+getData([
+  [bottomLeft.lat, bottomLeft.lng],
```

Comment:
Why not make this an object instead of a 2D array? It would make it a bit simpler / clear to pass through to `ST_MakeEnvelope`, right? It feels a bit clunky right now using the 2D array in the `containsClause` function as you have to reverse lat & lng for PostGIS.